modules/photosynthesis/DESCRIPTION

```diff
@@ -18,7 +20,14 @@ Description: The Predictive Ecosystem Carbon Analyzer (PEcAn) is a scientific
 Depends:
     rjags
 Imports:
-    coda (>= 0.18)
+    coda (>= 0.18),
+    utils,
+    graphics,
+    stats
+Suggests:
+    knitr,
+    rmarkdown,
+    markdown
 SystemRequirements: JAGS2.2.0
```

Comment:
@mdietze Should the JAGS requirement be updated to `>=`?  I'm currently testing with JAGS 4.3.0 and it all seems  to be working as advertised.

Reply:
Yeah, that's fine